app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -173,27 +173,31 @@ trait OnlineTestService {
     }
   }
 
-  private def inviteApplicant(application: OnlineTestApplication, token: String, userId: Int): Future[Invitation] = {
-    val scheduleId = getScheduleIdForApplication(application)
-    val inviteApplicant = buildInviteApplication(application, token, userId, scheduleId)
+  private def inviteApplicant(application: OnlineTestApplication, authToken: String, userId: Int, scheduleId: Int): Future[Invitation] = {
+
+    val inviteApplicant = buildInviteApplication(application, authToken, userId, scheduleId)
     cubiksGatewayClient.inviteApplicant(inviteApplicant).map { invitation =>
       audit("UserInvitedToOnlineTest", application.userId)
       invitation
     }
   }
 
-  private def emailInviteToApplicant(application: OnlineTestApplication, emailAddress: String, invitationDate: DateTime): Future[Unit] = {
-    val expirationDate = calculateExpireDate(invitationDate)
+  private def emailInviteToApplicant(application: OnlineTestApplication, emailAddress: String,
+    invitationDate: DateTime, expirationDate: DateTime): Future[Unit] = {
     val preferredName = application.preferredName
     emailClient.sendOnlineTestInvitation(emailAddress, preferredName, expirationDate).map { _ =>
       audit("OnlineTestInvitationEmailSent", application.userId, Some(emailAddress))
     }
   }
 
-  private def markAsCompleted(application: OnlineTestApplication)(onlineTestProfile: OnlineTestProfile): Future[Unit] =
-    otRepository.storeOnlineTestProfileAndUpdateStatusToInvite(application.applicationId, onlineTestProfile).map { _ =>
-      audit("OnlineTestInvitationProcessComplete", application.userId)
-    }
+  private def markAsCompleted(application: OnlineTestApplication)
+    (onlineTestProfile: Phase1TestProfile): Future[Unit] = for {
+    _ <- otRepository.insertPhase1TestProfile(application.applicationId, onlineTestProfile)
+    _ <- appRepository.updateProgressStatus(application.applicationId, ProgressStatuses.PHASE1_TESTS_INVITED)
+  } yield {
+      audit(s"ApplicationStatus set to ${ApplicationStatus.PHASE1_TESTS} - ProgressStatus set to" +
```

Comment:
There might be a reason we only use these one word event audits like 'OnlineTestInvitationEmailSent' - is there a reason you know of @ollyw for the one word audits rather than string messages like this?